### PR TITLE
fix: Change "Save" button naming

### PR DESCRIPTION
**Pull request by samuveth**
### Issues

Fixes https://github.com/snapshot-labs/snapshot/issues/4071

### Changes 

1. Change "Save" to "Apply changes"
2. Fix Treasury modal condition for button naming

image

### CODE REVIEW

File: src/components/ModalTreasury.vue [hunk 1]
```diff
@@ -95,7 +95,7 @@ watch(open, () => {
         primary
         @click="handleSubmit"
       >
-        {{ $t('save') }}
+        {{ treasury.name ? $t('applyChanges') : $t('add') }}
```
Comment (wa0x6e) [inline src/components/ModalTreasury.vue:98]: Shouldn’t this be `treasury?.name` , like the other checks ?

**Review by wa0x6e — CHANGES_REQUESTED** [review]

File: src/components/ModalTreasury.vue (hunk 1 above)
Comment (samuveth) [reply src/components/ModalTreasury.vue:98]: Not sure where you mean. Treasury name should always be an empty string at minimum as far as I can see. 
image

Comment (wa0x6e) [reply src/components/ModalTreasury.vue:98]: here https://github.com/snapshot-labs/snapshot/blob/9b0eee90c14337250b09c1aadf2a02d433cf96c8/src/components/ModalTreasury.vue#L44C20-L44C20

it's checking for treasury existence

Comment (samuveth) [reply src/components/ModalTreasury.vue:98]: Yeah that's not needed it this case. Will remove it

**Review by wa0x6e — APPROVED** [review]
utAck

**Pull request merged**